### ISSUE TITLE
Backup/Restore Problems

### ISSUE THREAD

**Issue by mbressman** [audreyt/ethercalc#702]
I believe I'm using Redis on an install I have running on Ubuntu 18.04 (via Hyper-V).  According to this post (https://github.com/audreyt/ethercalc/issues/62), the file containing all data is /var/lib/redis/dump.rdb

I've tried copying this file elsewhere (to test backup) and then deleting this file and then restarting EtherCalc or even rebooting virtual server, and yet each time when I restart EtherCalc and browse to a spreadsheet I'd previously created, it contains all previous data.

I also have a bunch of snapshot txt files and one audit txt file under a directory named "dump" under the home directory of the user I'm launching EtherCalc as - not sure what these are, but I've tried changing that directory to dump2 and still doesn't help me get rid of EtherCalc data.  

Basically what I'm aiming for here is to figure out where the EtherCalc data is stored (to eventually be able to back it up) and remove that file/location so that EtherCalc starts off as fresh (with no data), and then test putting back the backed-up file to see if I'm able to restore (and also to confirm I'll be backing up the correct file(s)).  Hope that makes sense...

Here's what I get when I launch EtherCalc (which I think confirms I'm running EtherCalc using Redis):
Please connect to: http://<SERVERNAME>:8000/
Starting backend using webworker-threads
Express server listening on port 8000 in development mode
Zappa 0.5.0 "You can't do that on stage anymore" orchestrating the show
Connected to Redis Server: localhost:6379

Any help is greatly appreciated!  Thanks!

**Comment by eddyparkinson** [comment]
Backup redis
---

from https://www.digitalocean.com/community/tutorials/how-to-back-up-and-restore-your-redis-data-on-ubuntu-14-04

```
$ls /var/lib/redis
You should see a dump.rdb file. That’s the Redis data. If appendonly is also enabled, you will also see an appendonly.aof or another .aof file, which contains a log of all write operations received by the server.
```

pasted from https://github.com/audreyt/ethercalc/issues/653#issuecomment-552689802

**Comment by mbressman** [comment]
Right, I see the dump.rdb file, but my problem is that I would assume if I remove that file then all the Redis data (and Ethercalc data) disappears, and yet when I remove that file and then relaunch EtherCalc (even after a reboot), data from spreadsheets that I've already entered is still there.  Which leads me to believe the data is being stored elsewhere...or perhaps I'm missing something?

**Comment by mbressman** [comment]
I think maybe my issue was that I wasn't stopping Redis
`sudo service redis-server stop`

But all good now, and the above link reposted by eddyparkinson is very helpful!  Thanks!
